api/v1/db/oracledb/developers-dao.js

```diff
@@ -41,7 +44,7 @@ const getDeveloperById = async (id) => {
     const sqlParams = {
       ID: id,
     };
-    const { rawDevelopers } = await connection.execute('select id from DEVELOPERS WHERE ID = :ID', sqlParams);
+    const { rawDevelopers } = await connection.execute('SELECT ID FROM DEVELOPERS WHERE ID = :ID', sqlParams);
```

Comment:
You actually used `rawDeveloper.id` in developers-serializer.js, so I think this `ID` field should have an alias `id` as you did before. Also, the bounded parameter `:ID` here should be camelCase as well.